data/xml/L14.xml

```diff
@@ -7607,7 +7607,7 @@
       <author><first>Emily</first><last>Franzini</last></author>
       <author><first>Greta</first><last>Franzini</last></author>
       <author><first>Monica</first><last>Lent</last></author>
-      <author><first>Maria</first><last>Moritz</last></author>
+      <author><first>Maria</first><last>Berger</last></author>
       <author><first>Simona</first><last>Stoyanova</last></author>
```

Comment:
The name here should match the PDF. If we want to merge an author page, we can create an entry in name_variants.